pyanaconda/bootloader.py

```diff
@@ -1917,6 +1917,14 @@ def __init__(self):
         super().__init__()
         self._packages64 = ["grub2-efi-aa64", "shim-aa64"]
 
+class ArmEFIGRUB(EFIGRUB):
+    _serial_consoles = ["ttyAMA", "ttyS"]
```

Comment:
should we attempt to get all the possible serial consoles in here?

Reply:
No, at least no in this patch series. I plan on having some more cleanup for ARM bits but as a separate PR, but the two above cover a lot of devices.